docs/user_guide/service_config.md

```diff
@@ -977,6 +977,37 @@ Example:
 admissionController.accessControl.externalGroups: "^sales$,^marketing$,^admin-"
 ```
 
+### Compresion setting
+
+The data in ConfigMap cannot exceed 1 MiB. YuniKorn supports Gzip algorithm to decompress data in the `binaryData` field.
```

Comment:
Nit: "YuniKorn supports **_the gzip_** algorithm" instead of "YuniKorn supports Gzip algorithm" (gzip is lowercased, and should have the article "the" preceding it).